front/src/components/Header/Header.vue

```diff
@@ -28,11 +29,12 @@ export default {
      * Set user name
      */
     setUserName(data){
-      if(data && data.nom && data.prenom)
```

Comment:
Uh?? Cela fonctionnait sans parenthèses???

Reply:
Yes ça fonctionne... mais c'est pas super propre...
https://stackoverflow.com/questions/4797286/are-braces-necessary-in-one-line-statements-in-javascript